src/server/dfly_main.cc

```diff
@@ -594,7 +595,20 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void print_basic_usage_info() {
+  std::cout << "* Default logdir is: \"/tmp/\". It can be overwritten with flag --log_dir=[path]\n";
+  std::cout << "* Logfile name format is: ";
+  std::cout << "dragonfly.[machine_name].[user_name].[WARNING|INFO|ERROR].[date and time]\n";
+  std::cout << "* For usage info flags type dragonfly [--help | --helpfull]\n";
+  const string df_website = "https://www.dragonflydb.io/";
```

Comment:
I would also skip the website part and lead directly to `https://www.dragonflydb.io/docs/` because we want to help our users here, not to market ourselves.